samples/show-extruded-graphics/src/main/java/com/esri/arcgismaps/sample/showextrudedgraphics/components/ShowExtrudedGraphicsViewModel.kt

```diff
@@ -0,0 +1,158 @@
+
+package com.esri.arcgismaps.sample.showextrudedgraphics.components
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.Color
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.PolygonBuilder
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.symbology.ExtrusionMode
+import com.arcgismaps.mapping.view.GraphicsOverlay
+import com.arcgismaps.mapping.view.SurfacePlacement
+import com.arcgismaps.mapping.view.Graphic
+import com.arcgismaps.mapping.symbology.SimpleFillSymbol
+import com.arcgismaps.mapping.symbology.SimpleFillSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleLineSymbol
+import com.arcgismaps.mapping.symbology.SimpleLineSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.symbology.RendererSceneProperties
+import com.arcgismaps.mapping.view.Camera
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.launch
+import kotlin.random.Random
+
+/**
+ * ViewModel for the ShowExtrudedGraphics sample.
+ *
+ * Prepares a 3D scene, adds an elevation source and a graphics overlay whose
+ * renderer is configured to extrude graphics based on the "height" attribute.
+ */
+class ShowExtrudedGraphicsViewModel(application: Application) : AndroidViewModel(application) {
+
+    // Create a camera and set the scene's initial viewpoint.
+    private val camera = Camera(
+        latitude = 28.4,
+        longitude = 83.0,
+        altitude = 20_000.0,
+        heading = 10.0,
+        pitch = 70.0,
+        roll = 0.0
+    )
+    // A 3D scene with a topographic basemap and an initial camera viewpoint.
+    val arcGISScene = ArcGISScene(BasemapStyle.ArcGISTopographic).apply {
+
+        initialViewpoint = Viewpoint(camera = camera, boundingGeometry = camera.location)
+
+        // Add a global elevation source to the base surface so extruded geometry has context.
+        baseSurface.elevationSources.add(
+            ArcGISTiledElevationSource(
+                uri = WORLD_ELEVATION_SERVICE_URL
+            )
+        )
+    }
+
+
+    // Graphics overlay that will contain extruded polygon graphics.
+    val graphicsOverlay: GraphicsOverlay
+
+    // Message dialog view model for reporting load errors.
+    val messageDialogVM = MessageDialogViewModel()
+
+    val squareSize = 0.01
+
+    init {
+        // Build renderer configured to extrude using the "height" attribute.
+        val outline = SimpleLineSymbol(
+            style = SimpleLineSymbolStyle.Solid,
+            color = Color.white,
+            width = 1f
+        )
+
+        val fillSymbol = SimpleFillSymbol(
+            style = SimpleFillSymbolStyle.Solid,
+            color = Color.fromRgba(220, 50, 50, 200),
```

Comment:
Let's make the fill symbols as solid symbols so the extruded graphics aren't overlayed over each other: 

```suggestion
            color = Color.fromRgba(220, 50, 50),
```

Reply:
Since the objective of sample is more about extrusion, we don't need to make them solid. I think the message of the sample would be conveyed, transparent or solid